cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractProgramTwillRunnable.java

```diff
@@ -155,13 +157,21 @@ public void initialize(TwillContext context) {
       CommandLine cmdLine = parseArgs(context.getApplicationArguments());
 
       // Loads configurations
-      hConf = new Configuration();
+      hConf = HBaseConfiguration.create();
+      String numClientRetries = hConf.get(Constants.HBase.CLIENT_RETRIES);
```

Comment:
What is this for? The whole runnable mechanism is based on the hConf serialized from the Master. Does that mean the hConf in master doesn't have these two properties set?? Anyway, if it has to be set programmatically, it should be set in the master side, right before it is localize the `hConf.xml`